### ISSUE TITLE
[QUESTION] Can I join to your team?

### ISSUE THREAD

**Issue by MESWEB** [denetwerkspecialist/Auctions#4]
@denetwerkspecialist can I join to your team and improve your project? It's seems like you have little bit problem with coding and English language.

**Comment by denetwerkspecialist** [comment]
Yes possible please send me a direct mail info@denetwerkspecialist.com

 

Van: MESWEB [mailto:notifications@github.com] 
Verzonden: vrijdag 8 november 2019 23:39
Aan: denetwerkspecialist/Auctions
CC: denetwerkspecialist; Mention
Onderwerp: [denetwerkspecialist/Auctions] [QUESTION] Can I join to your team? (#4)

 

@denetwerkspecialist <https://github.com/denetwerkspecialist>  can I join to your team and improve your project? It's seems like you have little bit problem with coding and English language.

**Comment by denetwerkspecialist** [comment]
Yes please send us a mail

 

Van: MESWEB [mailto:notifications@github.com] 
Verzonden: vrijdag 8 november 2019 23:39
Aan: denetwerkspecialist/Auctions
CC: denetwerkspecialist; Mention
Onderwerp: [denetwerkspecialist/Auctions] [QUESTION] Can I join to your team? (#4)

 

@denetwerkspecialist <https://github.com/denetwerkspecialist>  can I join to your team and improve your project? It's seems like you have little bit problem with coding and English language.

**Comment by denetwerkspecialist** [comment]
@MESWEB We are currently working on building in auctions. The intention is that the auctions are linked to the categories (Just like with BVA auctions). We encounter a number of problems that we cannot get the categories linked, because now only the url parameter id = is added. We want the following structure:

Currently:
https://auctions.dvdintermediair.nl/auction.php?id=82

Change to:
https://auctions.dvdintermediair.nl/auction.php?43972?category=15145563

Example:
Auction 1 Cars:
    - Mercedes (Category) (8)
       - Shows 8 lots from Mercedes
    - Audi (Category) (2)
        - Shows 2 lots from Audi
    - BMW (Category) (5)
        - Shows 5 lots from BMW

Auction 2 Cars:
    - Mercedes (Category) (4)
       - Shows 4 lots from Mercedes
    - Audi (Category) (1)
        - Shows 1 lots from Audi
    - BMW (Category) (4)
        - Shows 4 lots from BMW

If the users click on the category (Top menu) Mercedes then they must see 12 in 2 auctions.

See it here what we want:
https://www.bva-auctions.com/nl/auctions

We are looking for developers who can implement these functions in this OPEN SOURCE project.

Do you want to help with this project? mail to info@denetwerkspecialist.com

**Comment by MESWEB** [comment]
In this case:
https://auctions.dvdintermediair.nl/auction.php?43972?category=15145563
Category number is 15145563
But what's that number between? auction.php?43972?category

**Comment by denetwerkspecialist** [comment]
That's the number of the auction. Every auction has a unique number:
The auction number is here: 43972 and when you click inside the auction on a category then it append the category number to the url. Please check example below:

https://www.bva-auctions.com/nl/auction/index/37589?category=15146571

**Comment by denetwerkspecialist** [comment]
Can you send us an email, Because that's easier to talk.

**Comment by MESWEB** [comment]
Personaly for me link look's like that is ugly /auction/index/878787?
What's a reason for adding category number for item id?? More values in url make complicated life in future with editing system.
Customers don't care about category number in url. Google gonna get crazy when you change category of item. After that Google will index dead link item. My advice is don't mix item number with any category numbers.

**Comment by denetwerkspecialist** [comment]
It is not a matter of that want that kind of url,  we want to display a kind of selection of the items
If i is possible with a short URL fine but how?
Remember We got multiple auctions where sometimes items got the same category name

When i click on the front page on a category i want to show all items of the auction site, this works now on the front page.
But when i go to an auction (say Auctions 1) i want to show only the items of that Auction based on categories./Items
But when i go to an other auction (say Auctions 2) i want to show only the items of that Auction based on categories./Items

It does not matter hoe the url will be but how can in display or maybe filter on those categorie

**Comment by denetwerkspecialist** [comment]
any updates

**Comment by MESWEB** [comment]
PayPal integration is not work.
